src/desktop/apps/artwork/components/commercial/templates/inquire.jade

```diff
@@ -1,4 +1,8 @@
-if artwork.is_inquireable
+- var auctionPartner = artwork.partner.type === 'Auction' || artwork.partner.type === 'Auction House'
+- var newBuyNowFlow = user && user.hasLabFeature("New Buy Now Flow")
+
+//- TODO BNMO: Update to hide if the artwork is acquireable
+if artwork.is_inquireable && !((artwork.is_acquireable && auctionPartner) || newBuyNowFlow)
```

Comment:
This is the component that shows the "inquire" button on the sidebar of the artwork page.

You should see the inquire button if the artwork is `inquireable` _and_ you're not an auction partner with an `acquireable` work, and you aren't enabled for the lab feature.

Artworks that are part of auctions in any way don't show this component at all.